8.2/official/VERSION

```diff
@@ -0,0 +1 @@
+8.2.93-1c329ecd26
```

Comment:
Release versions drop the UniFi build-ID/commit tag, IE: this should be 8.2.93, not 8.2.93-1c329ecd26.

Reply:
I was not sure about this because the download links attached to the release message still contain the build-id/commit tag.. Thanks for resolving this issue! 👍